user/urls.py

```diff
@@ -6,6 +6,6 @@
 app_name = 'user'
 
 urlpatterns = {
-    path('/',),
+    path('',views.mainpage.as_view(),name = 'indexpage'),
```

Comment:
PEP8 conventions are missing - space after comma and you don't add spaces before and after while passing keyword arguments.
`path('', views.mainpage.as_view(), name='indexpage')`